IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CombinationInstrumentTransformer/DocPropertySet.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CombinationInstrumentTransformer" UniqueId="8f68a974-45e4-30c7-5db5-c91cafca08a3" ApplicableType="IfcFlowInstrument/COMBINED" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Do we need the combination Pset or can we just add the voltage and current ones to a combined PSet.

Reply:
As stated in my comment, these are awfully similar.